ray-curator/ray_curator/tasks/audio.py

```diff
@@ -0,0 +1,28 @@
+import os
+from dataclasses import dataclass
+
+from .tasks import Task
+
+
+@dataclass
+class DataEntry(Task[list]):
```

Comment:
```suggestion
class AudioClip(Task[list]):
```
or something similar? I was wondering what this Task will eventually encapsulate.

Reply:
It is pretty universal